number_parser/parser.py

```diff
@@ -250,6 +251,9 @@ def parse_number(input_string, language=None):
     if input_string.strip().isnumeric():
         return int(input_string)
 
+    if re.search("^(m{0,3})(cm|cd|d?c{0,3})(xc|xl|l?x{0,3})(ix|iv|v?i{0,3})$", input_string.lower()):
```

Comment:
Instead of lowercasing the input string, you can use `(?i)` at the very beginning of the pattern string to make it case-insensitive.